models/olids/conformed/conformed_patient_person.sql

```
@@ -6,25 +6,53 @@
+        patient_id,
+        person_id,
+        lds_business_id_person,
+        lds_source_record_id_person,
+        gp_practice_code,
+        lds_is_deleted,
+        lds_transform_datetime
+    FROM {{ ref('landing_patient_person') }}
+    -- the feed can ship duplicate rows per patient; keep the latest
+    QUALIFY ROW_NUMBER() OVER (
+        PARTITION BY patient_id
+        ORDER BY lds_transform_datetime DESC NULLS LAST, id
```

SQLFluff (4.2.2)

[error] Ambiguous order by clause. Order by clauses should specify order direction for ALL columns or NO columns.

(AM03)